README.md

```diff
@@ -274,3 +275,54 @@ rostopic pub -r 1 /flip_image std_msgs/Bool "{data: false}"
 The screenshot shows all the shell windows and their expected content:
 
 ![ROS 2 camera and ROS 1 rqt](doc/ros2_camera_ros1_rqt.png)
+
+## Example 3: run the bridge for AddTwoInts service
+
+example_interfaces/AddTwoInts service is already available in ROS2, so you should be able to run it after sourcing the setup script:
+
+```
+# Shell A:
+. <workspace-with-ros2>/install/setup.bash
+# Run client
+add_two_ints_client
+# Run server
+add_two_ints_server
+```
+
+For ROS you need to create that service by yourself using this tutorial:
+[Writing a Simple Service and Client](http://wiki.ros.org/ROS/Tutorials/WritingServiceClient(c%2B%2B)).
+The only change you need to do is to replace the package name from _beginner_tutorials_ to _example_interfaces_.
+When you are done you should be able to list this package using rossrv tool:
```

Comment:
When I'm done with mappings, it will be possible to connect ros_tutorials/roscpp_tutorials/TwoInts.srv with examples/example_interfaces/AddTwoInts.srv, so there will be no need to write any code